pod-master/podmaster.go

```diff
@@ -176,7 +186,16 @@ func main() {
 	validateFlags(&c)
 
 	machines := strings.Split(c.etcdServers, ",")
-	etcdClient := etcd.NewClient(machines)
+	var etcdClient *etcd.Client
+	if c.etcdCert != "" && c.etcdKey != "" && c.etcdCA != "" {
```

Comment:
Just test one here, you've already validated that if one is empty they all need to be empty.